controller/controller.go

```diff
@@ -186,6 +186,14 @@ func (c *Impl) Enqueue(obj interface{}) {
 	c.EnqueueKey(types.NamespacedName{Namespace: object.GetNamespace(), Name: object.GetName()})
 }
 
+// EnqueueSentinel returns a Enqueue method which will always enqueue a
+// predefined key instead of the object key.
+func (c *Impl) EnqueueSentinel(k types.NamespacedName) func(interface{}) {
+	return func(_ interface{}) {
```

Comment:
```suggestion
	return func(interface{}) {
```
I think this should work as wll.

Reply:
Yep. Done.